packages/shared/styles/components/SfSteps.scss

```diff
@@ -0,0 +1,103 @@
+@import "../variables";
+
+
+$sf-steps__active-line-height-mobile: 0.1875rem;
+$sf-steps__active-line-height-desktop: 0.125rem;
+$sf-steps__active-line-color: #1d1f22;
+$sf-steps__default-border: 0.0625rem solid #f1f2f4;
+
+$sf-steps__step-font-weight: 500;
+$sf-steps__step-font-size: 0.75rem;
+$sf-steps__step-font-color: $c-gray-secondary;
+
+
+%sf-store-active-line {
+  position: absolute;
+  content: "";
+  display: block;
+  height: $sf-steps__active-line-height-mobile;
+  width: 100%;
+  bottom: -$sf-steps__active-line-height-mobile /2;
+  left: 0;
+  background-color: $sf-steps__active-line-color;
+
+  @media screen and (min-width: $desktop-min) {
+    height: $sf-steps__active-line-height-desktop;
+    bottom: -$sf-steps__active-line-height-desktop /2;
+  }
+}
+
+.sf-steps {
+  &__steps {
+    width: 100%;
+    display: flex;
+    flex-flow: row nowrap;
+    justify-content: flex-start;
+    align-items: center;
+    border-bottom: $sf-steps__default-border;
+
+    &-step {
+      position: relative;
+      padding: 1.25rem 0;
+      font-size: $sf-steps__step-font-size;
+      color: $sf-steps__step-font-color;
+      font-weight: $sf-steps__step-font-weight;
+      text-transform: uppercase;
```

Comment:
a modifier in case they don't want it to be uppercase? @filrak @przspa wdyt?

Reply:
It will alter the feeling of the design quite a bit, but is easy to add @filrak @przspa ideas?